osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerTeamResults.cs

```diff
@@ -0,0 +1,96 @@
+using osu.Game.Screens.OnlinePlay.Multiplayer;
+using osu.Game.Users;
+
+namespace osu.Game.Tests.Visual.Multiplayer
+{
+    public class TestSceneMultiplayerTeamResults : ScreenTestScene
+    {
+        [TestCase(7483253, 1048576)]
+        [TestCase(1048576, 7483253)]
+        [TestCase(1048576, 1048576)]
+        public void TestDisplayWithTeams(int team1Score, int team2Score)
+        {
+            MultiplayerResultsScreen screen = null;
+
+            AddStep("show results screen", () =>
+            {
+                var rulesetInfo = new OsuRuleset().RulesetInfo;
+                var beatmapInfo = CreateBeatmap(rulesetInfo).BeatmapInfo;
+
+                var score = new ScoreInfo
+                {
+                    Rank = ScoreRank.B,
+                    TotalScore = 987654,
+                    Accuracy = 0.8,
+                    MaxCombo = 500,
+                    Combo = 250,
+                    Beatmap = beatmapInfo,
+                    User = new User { Username = "Test user" },
+                    Date = DateTimeOffset.Now,
+                    OnlineScoreID = 12345,
+                    Ruleset = rulesetInfo,
+                };
+
+                PlaylistItem playlistItem = new PlaylistItem
+                {
+                    BeatmapID = beatmapInfo.ID,
+                };
+
+                SortedDictionary<int, BindableInt> teamScores = new SortedDictionary<int, BindableInt>
+                {
+                    { 0, new BindableInt(team1Score) },
+                    { 1, new BindableInt(team2Score) }
+                };
+
+                Stack.Push(screen = new MultiplayerTeamResultsScreen(score, 1, playlistItem, teamScores));
+            });
+
+            AddUntilStep("wait for loaded", () => screen.IsLoaded);
+        }
+
+        [Test]
+        public void TestDisplayWithoutTeams()
+        {
+            MultiplayerResultsScreen screen = null;
+
+            AddStep("show results screen", () =>
+            {
+                var rulesetInfo = new OsuRuleset().RulesetInfo;
+                var beatmapInfo = CreateBeatmap(rulesetInfo).BeatmapInfo;
+
+                var score = new ScoreInfo
+                {
+                    Rank = ScoreRank.B,
+                    TotalScore = 987654,
+                    Accuracy = 0.8,
+                    MaxCombo = 500,
+                    Combo = 250,
+                    Beatmap = beatmapInfo,
+                    User = new User { Username = "Test user" },
+                    Date = DateTimeOffset.Now,
+                    OnlineScoreID = 12345,
+                    Ruleset = rulesetInfo,
+                };
+
+                PlaylistItem playlistItem = new PlaylistItem
+                {
+                    BeatmapID = beatmapInfo.ID,
+                };
+
+                Stack.Push(screen = new MultiplayerResultsScreen(score, 1, playlistItem));
+            });
+
+            AddUntilStep("wait for loaded", () => screen.IsLoaded);
+        }
```

Comment:
nit: this case seems to be pointless now that the team results variant have been separated to their own screens.

Reply:
The non-team-results variant will have no test coverage if this case is removed. So I'd rather keep it.

Reply:
Sure, in a separate test class as it's irrelevant from teams now.